components/suggest/src/provider.rs

```diff
@@ -12,19 +12,30 @@ use rusqlite::{
 
 use crate::rs::SuggestRecordType;
 
+/// Record types from these providers will be ingested when consumers do not
+/// specify providers in `SuggestIngestionConstraints`.
+pub(crate) const DEFAULT_INGEST_PROVIDERS: [SuggestionProvider; 6] = [
+    SuggestionProvider::Amp,
+    SuggestionProvider::Wikipedia,
+    SuggestionProvider::Amo,
+    SuggestionProvider::Yelp,
+    SuggestionProvider::Mdn,
+    SuggestionProvider::AmpMobile,
+];
```

Comment:
IMO consumers should always have to pass in providers they want to ingest. I don't think it makes sense for desktop to ingest AMP mobile by default, or vice versa. But that's orthogonal to this PR and here I'm just replacing `DEFAULT_RECORDS_TYPES` with `DEFAULT_INGEST_PROVIDERS`. It would be nice remove the concept of default ingest providers/record types in a follow-up.